test/demo/demo_unit.js

```diff
@@ -35,6 +35,9 @@ describe('Demo', () => {
 
   describe('config', () => {
     it('does not have entries for invalid config options', () => {
+      const exceptions = new Set()
```

Comment:
I assume this is because the `checkConfig` method does not identify these values.
Would it instead work if you added `array` to the `configPrimitives` variable inside `checkConfig` method?

Reply:
I tested it, but it doesn't work...